games/game.rb

```diff
@@ -0,0 +1,16 @@
+require '../item'
+class Game < Item
+  def initialize(publish_date, multiplayer, last_played_at)
+    super(publish_date)
```

Comment:
- [ ] Great work 👏 
I think that you will need to add the id and archived variables to the `super` here because when you are retrieving saved data back you may want to restore the object to it initial state.